roles/load-ycsb/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+user: ""
+elasticsearch:
+        server: ""
+        port: ""
```

Comment:
I've been struggling to get defaults to work the way I expect -- applying for missing, empty, or blank values. This method only takes care of missing, but empty or blank will be passed on as blank. Not a blocker problem at all, but something that could reduce user error if we can fix it.

Reply:
Ah, so this is due to me moving to passing the env vars via the container yaml. If the user does not pass the `elasticsearch` var ansible will throw up. I had to create these defaults... However, the SNAFU code should ignore empty value server/port.